datalake/src/main/resources/duties/7.2.16/aws/cdp_data_lake_medium_duty_with_profiler/medium_duty_ha.json

```diff
@@ -49,8 +49,8 @@
         "attachedVolumes": [
           {
             "count": 1,
-            "size": 250,
-            "type": "standard"
+            "size": 512,
+            "type": "gp2"
           }
         ]
       },
```

Comment:
Updates to AUXILIARY, GATEWAY, and master are not done.

Reply:
fixed